build/Tests/Tests.Shared/Tests.Shared.vcxitems

```diff
@@ -35,14 +35,61 @@
     <ProjectCapability Include="SourceItemsFromImports" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)\..\..\..\tests\functionaltests\Tests\$(PlatformTarget)\Default.AppxManifest.xml" />
+  </ItemGroup>
+  <ItemGroup>
     <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\..\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll" />
     <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)\..\..\..\Frameworks\CoreFoundation\CharacterSets\*" />
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)\..\..\..\tests\ContainerCRTDeps\$(ProcessorArchitecture)\*.dll" />
+    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)\..\..\..\tests\functionaltests\Tests\$(PlatformTarget)\Default.AppxManifest.xml" />
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\include\CppUtils.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\include\MockClass.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\tests\unittests\Tests.Shared\ByteUtils.h" />
   </ItemGroup>
   <Target Name="_GenerateRestoreGraphProjectEntry" />
+
+  <!-- This target adds reference to the Microsoft.UniversalCRT.Debug Extension SDK. This is that same thing that the normal Appx targets do but only for 
```

Comment:
Make sure to try the tests on ARM. Devices typically don't have the debug CRT installed to system32.

---
In reply to: [112344528](https://github.com/Microsoft/WinObjC/pull/2580#discussion_r112344528) [](ancestors = 112344528)